src/Utils/Utils.tsx

```diff
@@ -0,0 +1,3 @@
+export const getBaseUrl = () => "https://upskilling-egypt.com:3000";
+
+export const baseUrl = "https://upskilling-egypt.com:3000";
```

Comment:
I suggest creating .env file to store the base URL. This will be better if one of our endpoints depends upon a paid API. This means that we are making it available for everyone for free and we pay the bill.